.github/workflows/Format.yml

```diff
@@ -1,36 +1,12 @@
 name: Format
 
 on:
-  push:
-    branches:
-      - main
```

Comment:
Why not? What if someone pushes straight to `main` and then inside a PR someone else is surprised that the formatter fails